src/test/java/org/jitsi/meet/test/web/WebParticipantOptions.java

```diff
@@ -0,0 +1,111 @@
+package org.jitsi.meet.test.web;
+
+import org.jitsi.meet.test.base.*;
+
+import java.util.*;
+
+public class WebParticipantOptions
+    extends ParticipantOptions
+{
+    public static final String FAKE_AUDIO_PROP = "FAKE_AUDIO";
+    public static final String FAKE_VIDEO_PROP = "FAKE_VIDEO";
+    public static final String REMOTE_PROP = "REMOTE";
+    public static final String BINARY_PROP = "BINARY";
+    public static final String VERSION_PROP = "VERSION";
+
+
+    private static final String FAKE_AUDIO_FNAME_PROP
+        = "jitsi-meet.fakeStreamAudioFile";
+
+    private static final String FAKE_VIDEO_FNAME_PROP
+        = "jitsi-meet.fakeStreamVideoFile";
+
+    public WebParticipantOptions(Properties config)
+    {
+        super(config);
+
+        initDefaults();
+    }
+
+    private void initDefaults()
+    {
+        String fakeStreamAudioFile = config.getProperty(FAKE_AUDIO_FNAME_PROP);
+        if (fakeStreamAudioFile == null)
```

Comment:
For 'fakeStreamVideoFile' it also checks for blank, but here only for null. How about using org.apache.commons.lang3.StringUtils.isNotBlank for both ?